src/parser.js

```diff
@@ -6,7 +6,7 @@
  *
  */
 
-import createPlugins from './plugins';
+import createPlugins from './plugins'; // eslint-disable-line import/no-cycle
```

Comment:
Hva er konsekvensene av å ha sykliske avhengiheter?
Er det "trygt" å disable denne sjekken og bare la det være?

Reply:
Ikkje egentlig, men for å rendre visuelt element i forklaring var det enklest å kjøre en transformering fra en pågående transformering. Så det har vore slik ei stund uten at eslint har reagert på det.
Kan sjekke om eg kan skrive om dette med callbacks slik at importene ikkje er sykliske.

Reply:
Nei, eg er for dum til å skjønne korleis eg skal skrive om dette bedre. Koden er som sagt ikkje endra fra det som kjører i prod i dag, det er kun lintinga som er oppdatert.

Reply:
Jeg testa litt med å passe `transform` funksjonen videre fra `transform` -> 
 `getEmbedsFromHtml` -> `createPlugins` -> `createConceptPlugin`

Da får vi ikke linting feilen i alle fall, men om det er noe bedre aner jeg ikke. Kanskje noen som er flinkere enn meg på js har noen innspill?

Reply:
Oppdaterte koden med å sende med transform-funksjonen.